run/fillna_tier.py

```diff
@@ -477,21 +477,21 @@ def feature_engineering(score_file, tier_file):
     if np.all(pd.isna(patient['MutationAssessor_score'])):
         # Single variant query
         patient.loc[(pd.isna(patient['MutationAssessor_score'])) & np.array(indel_index), 'MutationAssessor_score'] = feature_stat.loc['MutationAssessor_score', '50%']
-        patient.loc[pd.isna(patient['MutationAssessor_score']), 'MutationAssessor_score'] = feature_stat.loc['MutationAssessor_score', '50%']
+        patient.loc[pd.isna(patient['MutationAssessor_score']), 'MutationAssessor_score'] = feature_stat.loc['MutationAssessor_score', 'min']
         patient['MutationAssessor_score'] = patient['MutationAssessor_score'].astype('float64')
     else:
         # Patient whole VCF
         patient['MutationAssessor_score'] = patient['MutationAssessor_score'].astype('float64')
         patient.loc[(pd.isna(patient['MutationAssessor_score'])) & np.array(indel_index), 'MutationAssessor_score'] = patient['MutationAssessor_score'].describe()['50%']
-        patient.loc[pd.isna(patient['MutationAssessor_score']), 'MutationAssessor_score'] = patient['MutationAssessor_score'].describe()['50%']
+        patient.loc[pd.isna(patient['MutationAssessor_score']), 'MutationAssessor_score'] = patient['MutationAssessor_score'].describe()['min']
 
 
     #ESP6500_AA_AF
     patient.loc[patient['ESP6500_AA_AF'] == '-', 'ESP6500_AA_AF'] = 0.0
     patient['ESP6500_AA_AF'] = patient['ESP6500_AA_AF'].astype('float64')
 
 
-    #ESP6500_AA_AF
+    #ESP6500_EA_AF
```

Comment:
An reason why the comments were changed?

Reply:
This is a tyoe lol